### DISCUSSION TITLE
Add Discord server to WiKi and update WiKi page.

### DISCUSSION THREAD

**Discussion by GeorgeLeithead** [CommunityToolkit/Maui#2032] [General]
Currently the .net MAUI Community Toolkit [WiKi page](https://github.com/CommunityToolkit/Maui/wiki), under 'Feedback and Requests' lists the dot-net Discord server and not the .[net MAUI Community Toolkit server](https://discord.gg/e6Nrh8aQ).  Can this be updated with a permanent link and also the copy updated to reflect as appropriate?

**Comment by ghost** [comment]
@GeorgeLeithead [net MAUI Community Toolkit server](https://discord.gg/e6Nrh8aQ) is expired. @jfversluis maybe you know more here?

**Reply by jfversluis** [reply]
Where did you find that link?

This one should work: https://discord.gg/pXfdCzfMBN

**Reply by ghost** [reply]
@jfversluis under Wiki page: https://github.com/CommunityToolkit/Maui/wiki

**Reply by jfversluis** [reply]
Oh wow that's some outdated info there 😄 thanks! Updated it!
